starlight/templates/views/home.html

```diff
@@ -13,6 +13,21 @@
 <div class="ui competency-container">
   <h1 class="ui black header">{% trans "Knowledge Table" %}</h1>
 
+<h1 class="ui black header">Filter by team: </h1>
+
+<form action="." method="post" class="ui small form" id="team_filter_form">
+    {% csrf_token %}
+    <div class="field">
+        <select class="ui selection dropdown" name="name" id="name">
+              <option value="All">All</option>
+              {% for key, teams_choice in form.fields.name.choices %}
+                  <option value="{{ key }}"> {{ teams_choice }}</option>
```

Comment:
It'd be cool if selecting the entry in the dropdown list automatically filtered the field, instead of having to click the Filter button afterwards.

We may need to do this via JS and a CSRF-free form.